tests/conftest.py

```diff
@@ -242,6 +242,7 @@ def default_conf(testdatadir):
         "db_url": "sqlite://",
         "user_data_dir": Path("user_data"),
         "verbosity": 3,
+        "strategy": "DefaultStrategy"
```

Comment:
SampleStrategy? DefaultStrategy should gone, I guess...

Reply:
this is for tests... we always used DefaultStrategy for tests - and i would continue to do so assuming that Samplestrategy may be modified by accident ... (happened to me, anyway) ... causing odd test failures :D

Reply:
Then this should be a TestStrategy (not loaded in the specific way by the resolver and not used in the specific way, like a fallback etc)?

Reply:
well it is - and it's not loaded as fallback - however it's accessible using `--strategy DefaultStrategy` - which i think is fine as the user had to select it explicitly.

Reply:
I did not understand -- will the DefaultStrategy be kept in freqtrade/optimize? If so and if the reason for this is the usage of it in the tests, should not it be 1) moved into the tests dir, 2) be renamed to TestStrategy or TestDefaultStrategy?

Reply:
i would leave it where it is - at least for now - to allow users to use it via `--strategy DefaultStrategy`. 
If it's an explicit selection by the user, then DefaultStrategy is fine - i just would like to avoid users running this strategy without thinking about it.